scripts/generate_expected_bpf2c_output.ps1

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation
```

Comment:
general comment on the function naming. I think these verbs are not approved. VSCode uses PSScriptAnalyzer(PSUseApprovedVerbs) and complains about non-approved verbs. Also the style for PS is verb-object such as Install-MyComponent etc.

I dont want to get into battle about the casing. The CICD scripts at least does not use snake casing.

Reply:
Changed to Verb-Object